docs-fr: make the Ingress TLS example match EN version (example URL mainly)

Horgix opened the pull request:
This PR is a minor update to the Ingress documentation in French: the page is talking about TLS everywhere, but the example DNS record used remains `sslexample.foo.com`. This PR simply changes it to `tlsexample.foo.com` for more consistency. Noticed it thanks to @franck-cussac !
 
As suggested in <https://kubernetes.io/docs/contribute/new-content/overview/#choose-which-git-branch-to-use> I tried to find a branch dedicated to the updates of the FR translation but didn't find any, so I'm filing this against master. Let me know if I should do something else!

Horgix commented:
I've signed the CLA now dear bot :slightly_smiling_face:

rekcah78 commented on content/fr/docs/concepts/services-networking/ingress.md:
````diff
@@ -328,7 +328,7 @@ metadata:
-Référencer ce secret dans un Ingress indiquera au contrôleur d'ingress de sécuriser le canal du client au load-balancer à l'aide de TLS. Vous devez vous assurer que le secret TLS que vous avez créé provenait d'un certificat contenant un CN pour `sslexample.foo.com`.
+Référencer ce secret dans un Ingress indiquera au contrôleur d'ingress de sécuriser le canal du client au load-balancer à l'aide de TLS. Vous devez vous assurer que le secret TLS que vous avez créé provenait d'un certificat contenant un CN pour `tlsexample.foo.com`.
 
 ```yaml
````
remplace all yaml block with this code like the EN page : 
{{< codenew file="service/networking/tls-example-ingress.yaml" >}}

Horgix replied:
Done

rekcah78 replied:
je voulais dire de remplacer tout le code du yaml par {{< codenew file="service/networking/tls-example-ingress.yaml" >}}

cela permet de toujours pointer sur l'exemple de référence.

rekcah78 commented on content/fr/docs/concepts/services-networking/ingress.md:
````diff
@@ -328,7 +328,7 @@ metadata:
 ```
 
-Référencer ce secret dans un Ingress indiquera au contrôleur d'ingress de sécuriser le canal du client au load-balancer à l'aide de TLS. Vous devez vous assurer que le secret TLS que vous avez créé provenait d'un certificat contenant un CN pour `sslexample.foo.com`.
+Référencer ce secret dans un Ingress indiquera au contrôleur d'ingress de sécuriser le canal du client au load-balancer à l'aide de TLS. Vous devez vous assurer que le secret TLS que vous avez créé provenait d'un certificat contenant un CN pour `tlsexample.foo.com`.
````
use https-example.foo.com like the EN page.

Horgix commented:
Glad to see you review this PR @rekcah78 :slightly_smiling_face: 

I added modifications according to your suggestions, and also added a small bit of text to better match the EN version - and renamed the PR accordingly.

Let me know if there is any other change you want me to do, if I need to squash my commits, etc. !

Horgix commented:
> je voulais dire de remplacer tout le code du yaml par {{< codenew file="service/networking/tls-example-ingress.yaml" >}}
> 
> cela permet de toujours pointer sur l'exemple de référence.

Got it! Didn't know that there was such template blocks in the site gen, pretty cool :slightly_smiling_face: 

Fixed it

Horgix commented:
Looks like the CI is failing because it's searching for the YAML file under the FR translation directory. What's the usual solution to this? I have a bunch in mind but I guess that it's something common here :slightly_smiling_face:

rekcah78 commented:
> Looks like the CI is failing because it's searching for the YAML file under the FR translation directory. What's the usual solution to this? I have a bunch in mind but I guess that it's something common here 🙂

You right, you need to copy the file in contents/fr/examples/ directory
[plus d'info ici](https://kubernetes.io/fr/docs/contribute/style/write-new-topic/#inclure-le-code-d-un-autre-fichier)

Horgix commented:
@rekcah78 well, I find it a bit weird to duplicate the file if the primary objective is to keep content coherent across all languages. Wouldn't it be better to reference the EN file in some way to avoid having to periodically "sync" the EN example file into the FR examples?

Anyway, updated the PR according to your suggestion :slightly_smiling_face:

rekcah78 commented:
/lgtm

Horgix commented:
What's the next step when it's marked as LGTM by a code owner? :slightly_smiling_face: 

It looks like I can't merge it myself. Let me know if I have anything to do here!

remyleone commented:
/lgtm
/approve

Horgix commented:
@remyleone any insights on the point/question I raised earlier in the comments? This one:

> @rekcah78 well, I find it a bit weird to duplicate the file if the primary objective is to keep content coherent across all languages. Wouldn't it be better to reference the EN file in some way to avoid having to periodically "sync" the EN example file into the FR examples?

remyleone commented:
You could have code snippets with comments in French, Chinese or Korean for instance. As the code snippets grow, comments are getting more and more useful to understand what is going on.
